tests/csm/rest/test_csm_users.py

```diff
@@ -675,37 +701,36 @@ def test_5005(self):
         """
         Test that GET API with empty value for sort_dir param returns 400 response code
         and appropriate error json data
-
         """
         test_case_name = cortxlogging.get_frame()
         self.log.info("##### Test started -  %s #####", test_case_name)
         test_cfg = self.csm_conf["test_5005"]["response_msg"]
         resp_error_code = test_cfg["error_code"]
         resp_msg_id = test_cfg["message_id"]
         resp_data = self.rest_resp_conf[resp_error_code][resp_msg_id]
-        msg = resp_data[0]
+        resp_msg_index = test_cfg["message_index"]
+        msg = resp_data[resp_msg_index]
         self.log.info(
             "Fetching the response for empty sort_by parameter with the expected status code")
         response = self.csm_obj.list_csm_users_empty_param(
             expect_status_code=const.BAD_REQUEST,
             csm_list_user_param="dir",
             return_actual_response=True)
-
+        self.log.info("Verifying the response status code")
+        assert response.status_code, const.BAD_REQUEST
```

Comment:
Above comment applicable

Reply:
will have a ticket to remove where we used commons\constants under our rest api tests